.github/workflows/ci.yml

```
@@ -58,8 +58,35 @@ jobs:
         run: mix format --check-formatted
 
       - name: Run Credo
-        run: mix credo
-        continue-on-error: false
+        id: credo
+        run: |
+          echo "## Credo Results" >> $GITHUB_STEP_SUMMARY
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)